web/src/main/java/org/cbioportal/web/parameter/PagingConstants.java

```diff
@@ -3,8 +3,8 @@
 public class PagingConstants {
```

Comment:
Probably should be combined with other staticly defined adjustable constants, or be made into properties that can be configured eternally to the code (portal.properties, etc). But this seems ok for a start. We should think whether or not people would actually want to tune these .